build.ps1

```diff
@@ -259,6 +260,7 @@ Foreach ($target in $targets) {
 write-host "Done creating packages."
 
 if ($buildType -eq 'stable') {
+    CreateRelease $version "ravendb" "ravendb" $env:vcsRootBranch $env:GITHUB_ACCESS_TOKEN $env:ravendbChangelog $DryRunVersionBump
```

Comment:
do we need any extra permissions on the token?

Reply:
According to github docs: `Users with push access to the repository can create a release.`
So I think we don't have to change anything.